### DISCUSSION TITLE
增大默认播放窗口大小和优化宽屏播放

### DISCUSSION THREAD

**Discussion by ghost** [BewlyBewly/BewlyBewly#1100] [Ideas]
- 这方面油管做的好，默认播放的窗口大，比目前Bilibili大不少。
- 此外Bilibili设置宽屏播放，视频下面内容会被挡住，需要**手动拉滚动条**。但是油管没有这个问题。（这个问题特别烦人）
- 油管是把视频标题和UP放在视频下方的。

**Comment by hakadao** [comment]
https://github.com/BewlyBewly/BewlyBewly/blob/main/README-cmn_CN.md
![image](https://github.com/user-attachments/assets/71b11944-afee-4dc3-9d90-7ce41083fd1c)
